File: xFit/UI/xfit_admin/lib/models/korisnik.dart
import 'package:json_annotation/json_annotation.dart';

part 'korisnik.g.dart';

@JsonSerializable()
class Korisnik {
  int? korisnikId;
  String? ime;
  String? prezime;
  DateTime? datumRodjenja;
  String? korisnickoIme;


  Korisnik(this.korisnikId, this.ime, this.prezime, this.datumRodjenja,this.korisnickoIme);

  factory Korisnik.fromJson(Map<String, dynamic> json) => _$KorisnikFromJson(json);

  Map<String, dynamic> toJson() => _$KorisnikToJson(this);
}

File: xFit_rabbit/xFit_rabbit/Models/Korisnik.cs
namespace xFit_rabbit.Models
{
	public class Korisnik
	{
		public int KorisnikId { get; set; }

		public string? Ime { get; set; }

		public string? Prezime { get; set; }

		public DateTime? DatumRodjenja { get; set; }

		public string? KorisnickoIme { get; set; }

		public int GradId { get; set; }

		public int SpolId { get; set; }
		public virtual ICollection<KorisnikUloga> KorisnikUlogas { get; } = new List<KorisnikUloga>();
	}
}
